build/android/src/main/java/net.minetest.minetest/MtNativeActivity.java

```diff
@@ -19,6 +19,10 @@ public void onCreate(Bundle savedInstanceState) {
 	public void onDestroy() {
 		super.onDestroy();
 	}
+	
+	@Override
+	public void onBackPressed() {
```

Comment:
I could do everything myself, but I'll be home in the evening. I'm in the hands of only the iphone and I'm in another city